src/OpenCensus/Utils/Arguments.cs

```diff
@@ -0,0 +1,42 @@
+﻿// <copyright file="Arguments.cs" company="OpenCensus Authors">
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of theLicense at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenCensus.Utils
+{
+    using System;
+
+    /// <summary>
+    /// Method for arguments validation.
+    /// </summary>
+    public class Arguments
```

Comment:
Let's not add non-essential public surface method

Reply:
Yep, it wasn't meant to be public - only internal. I'll fix this.